features/polyExplorer/src/context/explorer-context.jsx

```diff
@@ -141,6 +143,14 @@ export const ExplorerProvider = ({ children }) => {
         }
     }
 
+    function throwPopUp({ type, content }) {
```

Comment:
naming nit here too: "throw"PopUp calls "set"PopUp. But I would say what it really does is to set a popup of a specific type. So maybe `setPopUpWithType` or `setTypedPopUp`?
I don't like `throw` in general, anyway. It seems related to an error or exception. `create` might be a better option.